specfile/changelog.py

```diff
@@ -55,11 +55,13 @@ def __repr__(self) -> str:
     def extended_timestamp(self) -> bool:
         """Whether the timestamp present in the entry header is extended (date and time)."""
         try:
-            arrow.get(self.header, "ddd MMM DD YYYY")
+            arrow.get(self.header, "ddd MMM D YYYY")
```

Comment:
```suggestion
            arrow.get(self.header, ["ddd MMM D YYYY", "ddd MMM DD YYYY"])
```

Arrow allows passing a list of formats to try, the functionality should be equivalent but the code is simpler IMO.

Reply:
Thanks, that's definitely better.